setup.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+#Script to run To-do app using the bash script
+
+#Step1: Download package from the repository  
+sudo apt-get update -y
+
+#Step2: Install PyEnv Requisites using apt get
+sudo apt-get install -y git
+
+#Step3: Installs pip Python3 Version 
+sudo apt-get install python3-pip
+
+#Step4: Copy poetry installation file to local 
+curl -sSL https://raw.githubusercontent.com/python-poetry/ \
+poetry/master/get-poetry.py | python
+
+#Step5: Clone repository of To-Do App code from GIT
+git clone https://github.com/sunnygunjesh/DevOps-Course-Starter.git
+
+#Step6:Install poetry
+poetry install 
+
+#Step7: Copy required environment key/token for the app to run
```

Comment:
Worth noting that this creates the file, but it doesn't actually populate any variables - that's fine since we want those stored securely so we'll trust the developer to add them after this has run

Reply:
Thanks Jack, updated and pushed setup.sh per comments.Merging this to the main branch now.